src/templates/base.html

```diff
@@ -1,19 +1,26 @@
 <!DOCTYPE html>
 <html lang="en">
 
+{% load static %}
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="description" content="">
     <meta name="author" content="">
-    <link rel="icon" type="image/png" sizes="96x96" href="img/favicon.png">
+    <link rel="icon" type="image/png" sizes="96x96" href="{% static 'problems/img/favicon.png' %}">
 
-    <title>404 - Project Lovelace</title>
+    <title>Finding earthquake epicenters | Project Lovelace</title>
```

Comment:
Not sure what this file but if it's a base, shouldn't it get a different title? (Or delegate titles to other files?)